scrapbox/index.ts

```diff
@@ -7,34 +7,89 @@ import type {SlackInterface} from '../lib/slack';
 
 const getScrapboxUrl = (pageName: string) => `https://scrapbox.io/api/pages/tsg/${pageName}`;
 
+interface Link {
+	url: string;
+	domain: string; // e.g. scrapbox.io
+}
+
+interface ScrapboxUser {
+	id: string;
+	name: string;
+	displayName: string;
+	photo: string; // URL
+}
+
+interface ScrapboxLine {
+	id: string;
+	text: string;
+	userId: string;
+	created: number;
+	updated: number;
+}
+
+interface ScrapboxPage {
+	id: string;
+	title: string;
+	image: string; // URL
+	descriptions: string[];
+	user: ScrapboxUser;
+	pin: number;
+	views: number;
+	linked: number;
+	commitId: string;
+	created: number;
+	updated: number;
+	accessed: number;
+	snapshotCreated: number;
+	persistent: boolean;
+	lines: ScrapboxLine[];
+}
+
+export const scrapbox2slack = (s: string) => (
+	s.replace(/\[(https?:\/\/.+)\]/g, '$1') // 外部リンク
```

Comment:
[SHOULD] `[https://hoge.com] hoge [hoge]` とかが来たときにおかしくなる気がする

```suggestion
	s.replace(/\[(https?:\/\/.+?)\]/g, '$1') // 外部リンク
```

Reply:
ありがとうございます! 他にも最短一致で書くべきところがあったのでまとめて改良しました。